go/conversation/view_definition.py

```diff
@@ -583,9 +585,28 @@ def get(self, request, conversation):
         sio = StringIO()
         writer = csv.writer(sio)
         direction = request.GET.get('direction', 'inbound')
-        writer.writerows(conversation.get_aggregate_count(direction))
-        return HttpResponse(sio.getvalue(),
-            content_type='text/csv; charset=utf-8')
+        writer.writerows(self.get_aggregate_counts(conversation, direction))
+        return HttpResponse(
+            sio.getvalue(), content_type='text/csv; charset=utf-8')
+
+    def get_aggregate_counts(self, conv, direction):
```

Comment:
I disagree a bit -- the point of unit tests is to test small bits of functionality separately -- but I'm okay with this not being tested separately for now.